Assets/Scripts/Characters/Human/CustomCharacterManager.cs

```diff
@@ -402,7 +402,8 @@ private void setStatPoint(CreateStat type, int pt)
     private void Start()
     {
         int num;
-        QualitySettings.SetQualityLevel(5, true);
+        //IS THIS LINE OF CODE IMPORTANT??? IT AFFECTS GRAPHICS SCRIPTS
```

Comment:
When players open the character customization page in AoTTG, it displays them at the highest quality.

Reply:
It doesn't happen when the player opens the character customization page it happens when they spawn in.